source/_includes/puppet-collections/_gpg_command_warning.md

```diff
@@ -0,0 +1,4 @@
+    Could not find a valid trust path to the key.
+    gpg: WARNING: This key is not certified with a trusted signature!
+    gpg:          There is no indication that the signature belongs to the owner.
+    Primary key fingerprint: 47B3 20EB 4C7C 375A A9DA  E1A0 1054 B7A2 4BD6 EC30
```

Comment:
@nfagerlund Can includes be nested? If so, I might be able to simply include _rsk_fingerprint.md here instead of pasting in the updated fingerprint.